tests/main/postrm-purge/task.yaml

```
@@ -11,19 +11,34 @@ restore: |
     fi
 
 prepare: |
+    # TODO: unify this with tests/main/snap-mgmt/task.yaml
     echo "When some snaps are installed"
     #shellcheck source=tests/lib/snaps.sh
     . "$TESTSLIB"/snaps.sh
-    install_local test-snapd-tools
-    # services shall be stopped, even when using stop-command
-    install_local test-snapd-service
-    # socket units shall be cleaned up
-    install_local socket-activation
-    # as well as timers
-    install_local test-snapd-timer-service
-    # regular snaps and aliases too
-    snap install test-snapd-control-consumer
-    snap install test-snapd-auto-aliases
+
+    snap set core experimental.user-daemons=true
+
+    # Install a number of snaps that contain various features that have
+    # representation in the file system.
+    for name in test-snapd-service test-snapd-timer-service socket-activation \
+            test-snapd-user-service test-snapd-user-service-sockets \
+            test-snapd-user-timer-service test-snapd-tools \
+            test-snapd-control-consumer test-snapd-auto-aliases; do
+        install_local "$name"
+        snap list | MATCH test-snapd-service
```

Presumably you want to check more than just the first snap:
```suggestion
        snap list | MATCH "$name"
```

---

Lol, yes :)

---

Nah, we only care about test-snapd-service :rofl: